test/mpi/runtests

```diff
@@ -1218,7 +1218,8 @@ sub RunTestPassed {
         print TAPOUT "ok $g_total_seen - $workdir/$programname ${np} # time=$runtime\n";
     }
     if ($junitoutput) {
-        print JUNITOUT "    <testcase name=\"$g_total_run - $workdir/$programname ${np} ${progArgs} ${progEnv}\" time=\"$runtime\"></testcase>\n";
+        # Use %05d since the Junit plugin in Jenkins sort tests by name
+        printf JUNITOUT "    <testcase name=\"%05d - $workdir/$programname ${np} ${progArgs} ${progEnv}\" time=\"$runtime\"></testcase>\n", $g_total_run;
```

Comment:
The "testcase" line should also be changed in the functions, RunTestFailed and SkippedTest.

One additional request.  Can you add the "classname" attribute to the "testcase" line?
For example:
   my $str_group_name = $workdir;
   $str_group_name =~ s/.\///;
  printf JUNITOUT "    <testcase classname=\"$str_group_name\" name=\"%05d - $workdir/$programname ${np} ${progArgs} ${progEnv}\" time=\"$runtime\"></testcase>\n", $g_total_run;

This change would be needed in the functions: RunTestPassed, RunTestFailed and SkippedTest.

Reply:
Sure. It's done. Please review.